source/style-guide/nested-components.txt

```diff
@@ -0,0 +1,379 @@
+  contents in plain text, preceding the example with a line break and 
+  the text: "For example,".
+
+- If the example is a value in a code block, introduce the code block 
+  with "For example:". If more context follows the example, add "In the 
+  previous example,"" to the context.
+
+**Example:**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Before
+     - After
+
+   * - 
+       .. list-table::
+          :widths: 10 10 80
+          
+          * - 404
+            - Not Found
+            - The requested resource does not exist.
+
+          * - 405
+            - Method Not Allowed
+            - The HTTP method is not supported for the specified 
+              resource. Keep in mind that each resource may only 
+              support a subset of HTTP methods.
+
+              .. example::
+
+                 You are not allowed to ``DELETE`` the
+                 :opsmgr:`root </reference/api/root>` resource.
+
+     - 
+       .. list-table::
+          :widths: 10 10 80
+          
+          * - 404
+            - Not Found
+            - The requested resource does not exist.
+
+          * - 405
+            - Method Not Allowed
+            - The HTTP method is not supported for the specified 
+              resource. Keep in mind that each resource may only 
+              support a subset of HTTP methods.
+
+              For example, you are not allowed to ``DELETE`` the
+              :opsmgr:`root </reference/api/root>` resource.
+
+Procedures Inside Procedures
+----------------------------
+
+Remove the nested procedure and add its contents to an ordered list. 
+Start with a., b., c., d., and use i., ii., iii. for levels below that. 
+Do not nest steps more than that.
+
+**Example:**
+
+.. list-table::
+   :header-rows: 1
+
+   * - Before
+     - After
+
+   * - 
+       .. procedure::
+          :style: normal
+
+          .. step:: Open the Atlas Search index you want to update.
+      
+             .. procedure::
+                :style: connected
+         
+                .. step:: Navigate to your Atlas Search tab.
+         
+                .. step:: On the index you want to copy, click ``...`` in the :guilabel:`Action` column. 
+         
+                .. step:: Click :guilabel:`Edit With JSON Editor`.
+   
+          .. step:: Copy the index.
+     - 
+       .. procedure::
+          :style: normal
+
+          .. step:: Open the Atlas Search index you want to update.
+      
+             a. Navigate to your Atlas Search tab.
+             b. On the index you want to copy, click 
+                :icon-fa5:`ellipsis-h` in the :guilabel:`Action` column.
+             c. Click :guilabel:`Edit With JSON Editor`.
+   
+          .. step:: Copy the index.
+
+Tables Inside Tables
+--------------------
+
+Don't change tables inside tables until we have a better solution from 
+Design. You can complete and close the ticket without changing tables 
+inside tables.
```

Comment:
We can take this out. This made sense in the doc for fixing existing nested components, but we don't yet have a solution from design that's better than tables in tables, so we don't have any guidance to give for creating new pages. Once design gives us a solution we can add a section here to cover it, but until there's a better alternative writers can continue to nest them.